microservices/fe/cameraCV.py

```diff
@@ -16,7 +16,7 @@
         Examples of usage:
             1 - VideoStream() - this will run openning the camera with id=0 and will display and save frame
             2 - VideoStream(show_frame=False) - this will not display the windows with frame
-            3 - VideoStream(save_frame=False) - this will not save frame
+            3 - VideoStream(isSave_Frame=False) - this will not save frame
```

Comment:
We should keep the code style consistent. underscores instead of camelCase should probably be preferred here

Reply:
I cannot comment directly in VideoStream as this change was made prior to this PR, but if we want to keep consistent, we'd also change `framegrab` to `frame_grab`.

... And without forgetting to update documentation when we change this